packages/code-du-travail-frontend/src/outils/Components/DebugInfo.tsx

```diff
@@ -0,0 +1,18 @@
+type Props = {
+  data: Record<string, any>;
```

Comment:
Pour éviter le any :)

```suggestion
  data: Record<string, unknown>;
```